Sidney-/slugtrade
# Db populate refactor

New functionality:
- New command in manage.py: populate_db, which populates the database with users and items and an admin
- New script renew_db.sh: creates (or recreates) and populates database

How to test:

- Run script renew_db.sh, then start.sh to start the server.
- Go to route 'users' and verify that users 'admin' and 'test' plus 12 more random users exist.
- Log in as 'testuser' with password 'pass1234' and go to route 'profile/' to verify that all fields are populated.

- Log in to Django admin tools as 'admin' with password 'pass1234'.
- View table 'Users' to verify that username, first name, last name all exist for all regular users.
- View tables 'Items' to verify that each user posses one item.
- View 'Item images' to verify that the table contains entries.
- Click on one of the entries of 'Item images' to verify that the item's image has an image file extension.

Notes:
- There are many duplications of the profile images and item images sprinkled throughout the project file structure. I haven't been able to figure out which ones are being used and which can be deleted.
